src/programmatic-dmpu/web/index.html

```diff
@@ -0,0 +1,9 @@
+<div class="creative creative--programmatic-dmpu">
+  <a id="js-programmatic-dmpu" class="blink gs-container creative__link"
+      href="%%CLICK_URL_UNESC%%%%DEST_URL%%">
```

Comment:
Is it 4 × `%`? String templating is not easy to read 😆

Reply:
Yes, It is 4x %, it's an adamanger macro.

https://support.google.com/admanager/answer/2376981?hl=en#click&zippy=%2Cad-unit%2Cdestination-url%2Cfile-server%2Cheight-and-width%2Cpattern-match%2Csample-passback-tag-between-two-ad-manager-networks%2Csample-passback-tag-between-ad-manager-and-a-third-party-ad-server%2Ctarget-in-new-window%2Ctarget-window%2Ctag-for-child-directed-content%2Curi-encoding-for-creative-templates%2Ccachebuster-random-number%2Cclick%2Cexpand%2Cdevice-id-ios-android-roku-and-tvos%2Cpreview-mode%2Cscheme%2Csite%2Cviewed-impression%2Cvideo-description-url%2Cvideo-referrer-url%2Cvideo-duration-in-milliseconds%2Cvideo-id%2Cvideo-metadata%2Cvideo-title%2Ccustom-ad-id-freewheel%2Ciab-categories-freewheel%2Cgdpr%2Ctransparency-consent-tc-string%2Cadditional-consent-mode%2Ceea-and-uk-users-under-the-age-of-consent%2Cnon-personalized-ads

Reply:
Ah yes, I’m just thinking it would be nice to have these as an Enum or the like. I think it would be easy for someone to have one too few or too many.

Reply:
Ah yes sure. I think that's something that could be added in _shared to work across all the fabric formats maybe.

Can we merge this one, then I'll have a look at that as a separate pull request.